prisma/schema.prisma

```diff
@@ -86,37 +88,39 @@ model PrivateCloudRequestedProject {
   secondaryTechnicalLeadId String?              @db.ObjectId
-  secondaryTechnicalLead   User?                @relation("privateCloudRequestedSecondaryTechnicalLead", fields: [secondaryTechnicalLeadId], references: [id])
+  secondaryTechnicalLead   User?                @relation("privateCloudRequestedSecondaryTechnicalLead", fields: [secondaryTechnicalLeadId], references: [id], onDelete: Cascade)
   ministry                 Ministry
   cluster                  Cluster
   productionQuota          Quota
   testQuota                Quota
   developmentQuota         Quota
   toolsQuota               Quota
   commonComponents         CommonComponents
-  profileId                String?
   requestedProject         PrivateCloudRequest? @relation("privateCloudRequestedProject")
+  userRequestedProject     PrivateCloudRequest? @relation("privateCloudUserRequestedProject")
 }
 
 model PublicCloudRequest {
-  id                 String                      @id @default(auto()) @map("_id") @db.ObjectId
-  licencePlate       String
-  createdByEmail     String
-  decisionMakerEmail String?
-  type               RequestType
-  decisionStatus     DecisionStatus
-  humanComment       String?
-  active             Boolean                     @default(true)
-  created            DateTime                    @default(now())
-  decisionDate       DateTime?
-  projectId          String?                     @db.ObjectId
-  project            PublicCloudProject?         @relation("publicCloudProject", fields: [projectId], references: [id])
-  requestedProjectId String                      @unique @db.ObjectId
-  requestedProject   PublicCloudRequestedProject @relation("publicCloudRequestedProject", fields: [requestedProjectId], references: [id])
+  id                     String                       @id @default(auto()) @map("_id") @db.ObjectId
+  licencePlate           String
+  createdByEmail         String
+  decisionMakerEmail     String?
+  type                   PublicCloudRequestType
+  decisionStatus         DecisionStatus
+  humanComment           String?
+  active                 Boolean                      @default(true)
+  created                DateTime                     @default(now())
+  decisionDate           DateTime?
+  projectId              String?                      @db.ObjectId
+  project                PublicCloudProject?          @relation("publicCloudProject", fields: [projectId], references: [id], onDelete: Cascade)
+  requestedProjectId     String                       @unique @db.ObjectId
+  requestedProject       PublicCloudRequestedProject  @relation("publicCloudRequestedProject", fields: [requestedProjectId], references: [id], onDelete: Cascade)
+  userRequestedProjectId String                       @unique @db.ObjectId
+  userRequestedProject   PublicCloudRequestedProject? @relation("publicCloudUserRequestedProject", fields: [userRequestedProjectId], references: [id], onDelete: Cascade)
 }
```

Comment:
public cloud request also includes accountCoding, Budget

Reply:
The public cloud project and public cloud requested project includes accountCoding, Budget but not the request